services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/helpers/TopologyChecker.java

```diff
@@ -42,9 +42,9 @@
     public static boolean isIslEqual(TopologyDefinition.Isl linkDef, IslInfoData islInfoData) {
         PathNode discoveredSrcNode = islInfoData.getPath().get(0);
         PathNode discoveredDstNode = islInfoData.getPath().get(1);
-        return discoveredSrcNode.getSwitchId().equalsIgnoreCase(linkDef.getSrcSwitch().getDpId())
+        return discoveredSrcNode.getSwitchId().equals(linkDef.getSrcSwitch().getDpId())
```

Comment:
Would it be possible to use Objects.equals?

Reply:
I don't think that "null == null" is valid case here, so no need to use Objects.equals